### ISSUE TITLE
KLD and MMD weights and loss hyperparameters

### ISSUE THREAD

**Issue by ed-ortizm** [ed-ortizm/autoencoders#6]
Set KLD and MMD weights to 1 [computed using tf.reduce_mean] 

* Hyperparameters will be:
    *alpha
    *lambda
    *the MSE weight
